front/prisma/seed.ts

```diff
@@ -1,17 +1,28 @@
 import { PrismaClient } from "@prisma/client";
 import { hash } from "bcryptjs";
 
+const name = process.argv[2];
+const email = process.argv[3];
+const password = process.argv[4];
+
+if (!name || !email || !password) {
+    console.error(
+        `Missing argument:\nname: '${name}'\nemail: '${email}'\npassword: '${password}`,
+    );
+    process.exit(1);
+}
+
 const prisma = new PrismaClient();
 
 async function main() {
-    const password = await hash("test", 12);
+    const hashed = await hash("test", 12);
     const user = await prisma.omnilog_user.upsert({
```

Comment:
Are you handling the case where the user/email already exists? What should happen in this case?

Reply:
yes, with upsert! 

then the name and password is updated for that email

Reply:
Great. It looked like it was doing that, but I just wanted to check that this was the expected outcome 😊